web3/_utils/rpc_abi.py

```diff
@@ -17,16 +17,134 @@
 
 
 class RPC:
```

Comment:
@pipermerriam I was trying to find a more elegant solution to create these class variables which might eliminate some of the repetition, but I struggled to come up with anything simple b/c mypy doesn't actually run any code. Any suggestions?

Reply:
I think this is fine for now.